runtime/runtime/src/actions.rs

```diff
@@ -1127,6 +1129,22 @@ fn check_transfer_to_nonexisting_account(
     }
 }
 
+#[cfg(feature = "test_features")]
+fn apply_recorded_storage_garbage(
+    function_call: &FunctionCallAction,
+    state_update: &mut TrieUpdate,
+) {
+    if let Some(garbage_size_mbs) = function_call
+        .method_name
+        .strip_prefix("internal_record_storage_garbage_")
+        .and_then(|suf| suf.parse::<usize>().ok())
+    {
+        if state_update.trie.record_storage_garbage(garbage_size_mbs) {
+            tracing::warn!(target: "runtime", %garbage_size_mbs, "Generated storage proof garbage");
+        }
+    }
+}
+
 #[cfg(test)]
 mod tests {
 
```

Comment:
It'd be nice to include a script that can be used to submit the transactions that increase witness size.
Otherwise everyone will write their own scripts for their tests, which will be messy and not very reproducible.
Maybe a locust workload that reads the desired witness size as a commandline argument and submits the right transaction(s)?
It'd be nice to have a standardized way to do a witness size test on e.g. forknet.

Reply:
The approach is very straightforward and easy to use, so I don't think it would be hard to reproduce.
loctus workload can be added in a separate PR